### PR TITLE
Vagrant setup

### PR DESCRIPTION
**Pull request by darilldrems**
I have created a Vagrantfile with the shell provisioning script to setup for Ubuntu 14.04.

I also edited the README file to include vagrant installation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ http://tedx.amsterdam/
 
 Clone the repository and run `npm install`:
 
+##Vagrent Installation
```
Comment (Randynamic) [inline README.md:24]: Vagrent => Vagrant

File: README.md [hunk 2]
```diff
@@ -21,6 +21,13 @@ http://tedx.amsterdam/
...
+Clone the repository and run vagrant up then vagrant ssh
```
Comment (Randynamic) [inline README.md:25]: In documentation if you refer to code or commands please wrap them in backticks (`) for better readability e.g. `vagrant up` or `vagrant ssh`

File: README.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
```
Comment (Randynamic) [inline README.md:2]: +1 for writing docs Thanx :+1:

**Comment by Randynamic** [comment]
I'll look into the working pieces tonight looks very good so far

**Comment by darilldrems** [comment]
Okay. Thanks for the feedback. Will adhere to wraping commands in `` going forward.

**Pull request merged**